entries/o/o2switch.fr.json

```
@@ -0,0 +1,14 @@
+{
+    "o2switch": {
+        "domain":"o2switch.fr",

```

```suggestion
        "domain": "o2switch.fr",
```

Add space between the key and value